### PR TITLE
[WARNING] Warn when extending JSONAPISerializer with extractMeta

### PR DESCRIPTION
**Pull request by wecc**

### CODE REVIEW

File: addon/serializers/json-api.js [hunk 1]
```diff
@@ -93,6 +93,39 @@ var dasherize = Ember.String.dasherize;
...
+
+  One scenario would be to camelCase the meta keys of your payload. The example
+  below shows how this could be done using `normalizeArrayResponse` and
+  `extractRelationship`.
+
+  ```app/serializers/application.js
+  export default JSONAPISerializer.extend({
+
+    normalizeArrayResponse(store, primaryModelClass, payload, id, requestType) {
+      let normalizedDocument = return this._super(...arguments);
```
Comment (urbany) [inline addon/serializers/json-api.js]: Good example. Just a nitpick: shouldn't have a return here, should be:
`let normalizedDocument = this._super(...arguments);`

Same thing for `extractRelationship`

**Comment by wecc** [comment]
@bmac Sounds 👍 PR/commit updated!

**Comment by pangratz** [comment]
There's a `'` missing in https://github.com/emberjs/data/pull/4603/files#diff-c939dc9dc1a354cd6a97bd2c4ab65333R785

Wanna fix that in this PR too?

**Comment by wecc** [comment]
@pangratz done!

**Pull request merged**